weekly-standup-template.md

```diff
@@ -79,16 +74,36 @@ Other issues can be dealt with in coworking time.
 
 ### Notes
 
-- Summary of discussion points
-- Use subsections if appropriate
+#### Blockers
+
+- Blocker
+  - Address the blocker
+
+#### Coworking session suggestions
+
+- Suggestion
+- Suggestion
 
-### Decisions
+#### Plans for the week
```

Comment:
Good point. Agreed that putting "I'm working on Story X, as I have been for the past two months" is not a good use of time.

Maybe we need to wait and see how we use this section?

I was imagining this would be used for smaller tasks, engagement.
It might be better to keep the story level allocation in the project board and not duplicate it here?

Reply:
let's focus this on taks level issues yes, and see what happens keeping an eye on that. even at task level we may find tha tour usual issue is longer than a week

Reply:
Attempted to improve this in ac1eb64